torchvision/models/mobilenet.py

```diff
@@ -108,7 +102,7 @@ def forward(self, x):
         return x
 
 
-def mobilenet_v2(pretrained=False, progress=True, **kwargs):
+def mobilenet_v2(pretrained=False, progress=True, inverted_residual_setting=None, **kwargs):
```

Comment:
Can you add another test passing a different configuration, with fewer lines in the inverted_residual_setting?

Reply:
added a test in test_models